### PR TITLE
Pandas.io.formats.style.Styler.pipe docstring PR02

### PR DESCRIPTION
**Pull request by Genza999**
Solves:

- Unknown parameters {*args, **kwargs :}

pandas.io.formats.style.Styler.pipe: Unknown parameters {*args, **kwargs :} in #27976

### CODE REVIEW

File: pandas/io/formats/style.py [hunk 1]
```diff
@@ -1350,8 +1350,11 @@ def pipe(self, func, *args, **kwargs):
             Function to apply to the Styler.  Alternatively, a
             ``(callable, keyword)`` tuple where ``keyword`` is a string
             indicating the keyword of ``callable`` that expects the Styler.
-        *args, **kwargs :
-            Arguments passed to `func`.
+        *args : iterable, optional
+            Positional arguments passed into ``func``.
+        **kwargs : dict, optional
+            A dictionary of keyword arguments passed into ``func``.
+
```
Comment (TomAugspurger) [inline pandas/io/formats/style.py]: This line may be causing issues.

**Comment by WillAyd** [comment]
lgtm except merge conflict - can you fix up?

**Comment by Genza999** [comment]
> lgtm except merge conflict - can you fix up?

Hey @WillAyd , I happened to implement the changes in this pull request in another pull request that I implemented a number of changes and it got merged before this one. For this reason I was wondering if its okay to close this PR instead.

**Comment by WillAyd** [comment]
Sure no problem

**Pull request closed without merging**